controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -476,10 +476,11 @@ func (r *StorageConsumerReconciler) reconcileCephFilesystemSubVolumeGroup(
 	}
 
 	svg := &rookCephv1.CephFilesystemSubVolumeGroup{}
-	svg.Name = subVolumeGroupName
+	svg.Name = fmt.Sprintf("%s-%s", cephFs.Name, subVolumeGroupName)
```

Comment:
This is not the agreed design. There is only one subvolumegroup per consumer and its name should be by default the consumer name unless it got override by a name from an pre configured configmap name

Reply:
_**its name should be by default the consumer name**_ isn't this statement for the Ceph layer? we are also giving similar treatment to rns, the one mentioned in the config is at ceph and we prepend blockpool name at rook.